nx/lib/io/nx_io.mli

```diff
@@ -191,6 +191,25 @@ val save_npz : ?overwrite:bool -> string -> (string * packed_nx) list -> unit
 
     Save multiple named nxs to a NumPy `.npz` archive. *)
 
+(** {1 Text Format} *)
+
+val save_txt : ?sep:string -> ?append:bool -> out:string -> ('a, 'b) Nx.t -> unit
```

Comment:
NumPy has more options: https://numpy.org/doc/2.3/reference/generated/numpy.savetxt.html

Without being prescriptive, you might want to check if there's options from there that we want to support